pvr.nextpvr/addon.xml.in

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <addon
   id="pvr.nextpvr"
-  version="21.0.3"
+  version="21.0.4"
```

Comment:
Usually we increase the minor version number when we introduce new features, not the micro version.

Reply:
Fair enough, I considered it a minor change, but I have changed it.  All the .0 releases for new changes could look funny.

Reply:
New feature => increase minor, set micro to zero
Bug fix only => increase micro 

This is how many software projects handle it. So do we.